modules/govuk/manifests/app.pp

```diff
@@ -359,6 +373,13 @@
       fields  => {'application' => $title},
     }
 
+    @filebeat::prospector { "${title}-app-err":
+      ensure => $ensure,
+      paths  => "/var/log/${title}/app.err.log",
+      tags   => ['stderr', 'app'],
```

Comment:
we should pass a `json` parameter here (and elsewhere in this PR) to match the `govuk_logging::logstream` instance